components/MembersList/requests.jsx

```diff
@@ -10,9 +10,22 @@ export const fetchVeolasBalance = async ({ account }) => {
   return balance;
 };
 
+export const fetchVotingPower = async ({ account }) => {
+  const contract = getDelegateContributeContract();
+  const votingPower = await contract.methods.votingPower(account).call();
+  return votingPower;
+};
+
 export const checkVeolasThreshold = async (account, thresholdInWei) => {
   const balance = await fetchVeolasBalance({ account });
   const bNBalance = BigNumber.from(balance);
   const processedThreshold = BigNumber.from(thresholdInWei);
   return bNBalance.gte(processedThreshold);
 };
+
+export const checkVotingPower = async (account, thresholdInWei) => {
+  const votingPower = await fetchVotingPower({ account });
+  const bNBalance = BigNumber.from(votingPower);
```

Comment:
tiny, I guess copypasted - should be bNVotingPower

Reply:
done!